.make.versions

```diff
@@ -51,4 +51,7 @@ CODE_QUALITY_RAY_VERSION=0.4.0$(RELEASE_VERSION_SUFFIX)
 INGEST_TO_PARQUET_VERSION=0.4.0$(RELEASE_VERSION_SUFFIX)
 INGEST_TO_PARQUET_RAY_VERSION=0.4.0$(RELEASE_VERSION_SUFFIX)
 
+LICENSE_CHECK_PYTHON_VERSION=0.4.0$(RELEASE_VERSION_SUFFIX)
```

Comment:
should we need to have common license value. or can it start from 0.1.0 version

Reply:
following transform conventions